conf/modules.config

```diff
@@ -1126,37 +1140,68 @@ if (!params.skip_multiqc) {
 }
 
 //
-// Salmon pseudo-alignment options
+// Salmon/ Kallisto pseudo-alignment options
 //
 
-if (!params.skip_pseudo_alignment && params.pseudo_aligner == 'salmon') {
+if (params.pseudo_aligner == 'salmon') {
     process {
-        withName: '.*:QUANTIFY_SALMON:SALMON_QUANT' {
+        withName: '.*:QUANTIFY_PSEUDO:SALMON_QUANT' {
             ext.args   = params.extra_salmon_quant_args ?: ''
             publishDir = [
                 path: { "${params.outdir}/${params.pseudo_aligner}" },
                 mode: params.publish_dir_mode,
                 saveAs: { filename -> filename.equals('versions.yml') || filename.endsWith('_meta_info.json') ? null : filename }
             ]
         }
+    }
+}
+
+if (params.pseudo_aligner == 'kallisto') {
+    process {
+        withName: '.*:QUANTIFY_PSEUDO:KALLISTO_QUANT' {
+            ext.args = {
+                [
+                    meta.single_end ?
+                        "--single " +
+                        "--fragment-length ${meta.fragment_len ?: params.kallisto_default_fraglen} " +
```

Comment:
Might be better to have an input parameter to the module that sets `fragment_length`? Think we should try and avoid this sort of logic in configs if possible.

Reply:
The fragment length is non-mandatory in paired-end mode, so I'd put it in ext.args (again, per nf-core standards as I've understood them). But then again it IS mandatory in single-end, so I guess it's a weird case. 

I'll PR to the modules repo.

Reply:
Waiting on https://github.com/nf-core/modules/pull/4309 and maintainer feedback